src/heads-up/components/property-data.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <span>{{ tooltipMessage }}</span>
+</template>
+<script>
+
+function isDefined(tag) {
```

Comment:
Put these functions inside of the vue cycle by placing them in the `methods` object (place that below the `computed`) and then call the functions with `this` in front of it.